doc/admin-guide/files/records.config.en.rst

```diff
@@ -1194,6 +1194,11 @@ Parent Proxy Configuration
 
    The amount of time allowed between connection retries to a parent cache that is unavailable.
 
+.. ts:cv:: CONFIG proxy.config.http.parent_proxy.max_trans_retries INT 4
+
+   Limits the number of simultaneous transactions that may retry a parent once the parents
```

Comment:
Is this for a single object (if using consistent_hash), or for all objects to that parent?

Reply:
It's for all transactions to that parent on that parent.config line.